plat/common/include/x86/paging.h

```diff
@@ -266,6 +268,15 @@ pgarch_init(void)
 	}
 #endif /* PT_LEVELS == 5 */
 
+	/* Check for PAT support */
+	ukarch_x86_cpuid(0x1, 0, &eax, &ebx, &ecx, &edx);
+	if (unlikely(!(edx & X86_CPUID1_EDX_PAT))) {
+		uk_pr_crit("Page table attributes are not supported.\n");
+		return -ENOTSUP;
+	}
+	/* Write default value to MSR, in case BIOS modified them */
+	wrmsrl(0x277, 0x0007040600070406);
```

Comment:
Where is the default value provided to MSR determined, is there a reference you can site?

Reply:
Good point, I will add a reference to the comment.